homeassistant/components/telegram_bot/entity.py

```diff
@@ -0,0 +1,31 @@
+"""Base entity for Telegram bot integration."""
+
+import telegram
+
+from homeassistant.const import CONF_PLATFORM
+from homeassistant.helpers.device_registry import DeviceEntryType, DeviceInfo
+from homeassistant.helpers.entity import Entity
+
+from . import TelegramBotConfigEntry
+from .const import DOMAIN
+
+
+class TelegramBotEntity(Entity):
+    """Base entity."""
+
```

Comment:
```suggestion
    _attr_has_entity_name = True
```
should be set for all new entities (you might need to set the `name` of the `notify` entity explicitly to `None` then)